.gitignore

```diff
@@ -16,4 +18,6 @@ work
 
 key.asc
 .~*
-VERSION*
\ No newline at end of file
+VERSION*
+
+test/test_data/*.xlsx
```

Comment:
This line is added to avoid tracking the generated test outputs.

Reply:
Just adding this now...